bufferapp/buffer-publish
# Display only available products on App Shell

## Description

This PR includes the changes to:
- Upgreade `@bufferapp/ui` to 5.34.2.
- Pass a list of `featureFlips` to the `AppShell`.
- If the user has `reply` on the list of `productLinks` then set the `enableReply` feature flip.
- Add tests.
- Update snapshots.

## Context & Notes

This changes are required in order to remove the `Reply` product link from the AppShell for users that do not have access to that product already.

## Steps to QA

1) Login with an account that has access to Analyze and Reply.
2) Confirm that Publish, Analyze and Reply are displayed on the AppShell and all 3 links work.
3) Login with an account that has access to Analyze but it does not have access to Reply.
4) Confirm that only Publish and Analyze are visible on the AppShell.

## Checklist



-   [ ] My code follows the code style and guidelines of this project. 
-   [ ] I have tested different user stories. 
-   [ ] I have identified similar or related features and tested they are still working.
-   [x] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`
